mediacenter/models.py

```diff
@@ -0,0 +1,78 @@
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ideasbox.models import TimeStampedModel
+from search.models import SearchableQuerySet, SearchMixin
+from .utils import guess_type
+
+
+class DocumentQuerySet(SearchableQuerySet, models.QuerySet):
+    def image(self):
+        return self.filter(type=Document.IMAGE)
+
+    def video(self):
+        return self.filter(type=Document.VIDEO)
+
+    def pdf(self):
+        return self.filter(type=Document.PDF)
+
+    def text(self):
+        return self.filter(type=Document.TEXT)
+
+    def audio(self):
+        return self.filter(type=Document.AUDIO)
+
+
+class Document(SearchMixin, TimeStampedModel, models.Model):
```

Comment:
If it inherites from model.Model, you don't need to use model.Model when defined your model here. Your choice, but either you use it as a model subclass, or a mixin, but the name and usage should be consistent